test_runner/src/main/kotlin/ftl/gc/android/SetupAndroidTest.kt

```diff
@@ -9,5 +9,7 @@ internal fun TestSpecification.setupAndroidTest(config: AndroidTestConfig) = app
             androidInstrumentationTest = createAndroidInstrumentationTest(config)
         is AndroidTestConfig.Robo ->
             androidRoboTest = createAndroidRoboTest(config)
+        is AndroidTestConfig.GameLoop ->
+            androidTestLoop = createGameLoopTest(config)
```

Comment:
this is very short and I deb that it could be a single line :)

Reply:
👍